installer/x86_64/install.sh

```diff
@@ -485,7 +514,14 @@ fi
 
 # Add a menu entry for the DEMO OS
 # Note: assume that apparmor is supported in the kernel
-demo_grub_entry="$demo_volume_revision_label"
+demo_grub_entry="$demo_volume_revision_label-${git_revision}"
+if [ "$environment" = "sonic" ]; then
+    running_sonic_revision=$(cat /etc/sonic/sonic_version.yml | grep build_version | cut -f2 -d" ")
```

Comment:
I'm checking for currently running version in order to keep it. It could happen currently running is not the default one